src/Build/BackEnd/Components/RequestBuilder/RequestBuilder.cs

```diff
@@ -824,17 +815,7 @@ private async Task BuildAndReport()
 
                 thrownException = ex;
             }
-            catch (LoggerException ex)
-            {
-                // Polite logger failure
-                thrownException = ex;
-            }
-            catch (InternalLoggerException ex)
-            {
-                // Logger threw arbitrary exception
-                thrownException = ex;
-            }
-            catch (Exception ex)
+            catch (Exception ex) // LoggerException is a polite logger failure. InternalLoggerException is an arbitrary exception. Handle them the same.
```

Comment:
I don't understand the "handle them the same" part of this comment. Maybe just delete the comment?